### ISSUE TITLE
[Bug] sfp prepare error

### ISSUE THREAD

**Issue by nkskaare** [flxbl-io/sfp#25]
**Describe the bug**
`sfp prepare`  command error

**To Reproduce**
run `sfp prepare` in the latest feburary release of sfp-rc:alpha / sfp-rc:release-Feb24

**Expected behavior**
Expect the command to run as before the update

**Screenshots**
![Screenshot 2024-02-27 at 10 10 08](https://github.com/flxbl-io/sfp/assets/35693870/6ddd8234-7d4d-4bff-975d-996501483192)

**Platform Details (please complete the following information):**
 - OS: Ubuntu
 - Version: 36.0.5 (Release Feb24)
 - Salesforce CLI(sfdx cli) Version: -
 - CI Platform: Github Actions

**Additional context**
 ```
command: prepare
    Error: Unable to execute command .. Error: /usr/lib/node_modules/@flxblio/
    resources/schemas/pooldefinition.schema.json: ENOENT: no such file or 
    directory, open '/usr/lib/node_modules/@flxblio/resources/schemas/pooldefi
    nition.schema.json'
```

**Comment by azlam-abdulsalam** [comment]
Thanks @nkskaare , will be fixing it shortly,

**Comment by azlam-abdulsalam** [comment]
This is fixed in 36.0.6
